OpenMage/magento-lts
# Add loading delay adminhtml

### Description (*)

As described in #2419 it can be annoying on a fast connection that the admin dashboard "Please wait" message shows up for very fast AJAX requests.

This PR sets a timeout on showing the loading indicator so that it only shows up after 100ms. If the request is quicker than that, the indicator will not show.

I also removed some Internet Explorer stuff...

Paging @ADDISON74 please test :)

Possible TODO, allow this value to be configurable? I'm not really sure it's needed and would cause some extra complexity because we need to output the variable value in a phtml file somewhere so that we can read it in JS. The default of 100ms is a pretty commonly used debounce value.

### Related Pull Requests


### Fixed Issues (if relevant)

1. #2419 

### Manual testing scenarios (*)

1. Load some quick operation in the Magento dashboard like filter customers by email `foobar@example.com`. The indicator should not show up if your server is fast (especially if its local)
2. Load some longer operation in the Magento dashboard like filter customers by name `Foo Bar`. The indicator should show up if you have a good amount of customers.

### Questions or comments


### Contribution checklist (*)
